frontend/src/main/java/com/google/tripmeout/frontend/error/WrongFormatUriException.java

```diff
@@ -0,0 +1,20 @@
+package com.google.tripmeout.frontend.error;
+
+import com.google.tripmeout.frontend.error.TripMeOutException;
+import java.lang.Throwable;
+import javax.servlet.http.HttpServletResponse;
+
+public class WrongFormatUriException extends TripMeOutException {
+  public WrongFormatUriException(String errorMsg) {
+    super(errorMsg);
+  }
+
+  public WrongFormatUriException(String errorMsg, Throwable cause) {
+    super(errorMsg, cause);
+  }
+
+  @Override
+  public int restStatusCode() {
+    return HttpServletResponse.SC_BAD_REQUEST;
```

Comment:
Posted about it in the chat for faster turn around, but this probably isn't what we want to do.